jsonschema-core.xml

```diff
@@ -2416,7 +2858,8 @@ User-Agent: product-name/5.4.1 so-cool-json-schema/1.0.2 curl/7.43.0
                 Brad Bowman,
                 Gowry Sankar,
                 Donald Pipowitch,
-                and Dave Finlay
+                Dave Finlay,
+                and Greg Dennis
```

Comment:
You have an author credit, we don't really need to put you in the acknowledgements :-)

Reply:
that's fair.  I don't see you there...  ✔️